libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/mysql/MySQLNoGreaterThan5740SchemaAccessor.java

```diff
@@ -594,7 +599,6 @@ public Map<String, List<DBTableIndex>> listTableIndexes(String schemaName) {
                 columnNames.add(rs.getString("COLUMN_NAME"));
                 index.setColumnNames(columnNames);
                 index.setGlobal(true);
-                handleIndexAvailability(index, rs.getString("COMMENT"));
```

Comment:
why this code is deleted?

Reply:
I consulted kernel classmates and found that the `comment` field of the `information_schema.STATISTICS` table and the `comment` field of the `show index from` query come from different sources and have different meanings. The `comment` field of `information_schema.STATISTICS` cannot be used to indicate the status of the index, so for `Map<String, List<DBTableIndex >> listTableIndexes(String schemaName); `method does not query whether the index is available. Actually only the `List<DBTableIndex> listTableIndexes(String schemaName, String tableName) `method needs to provide the available status of the index.